README.md

```diff
@@ -141,11 +143,12 @@ src="assets/linktree.webp" style="width:15px;height:15px;"> [Linktree](https://l
 | [NodeStake](https://nodestake.top/)    | <https://nodestake.top/okp4>                       | ✅    |
 | [AVIAONE](https://aviaone.com)         | <https://aviaone.com/blockchains-service/okp4-testnet-okp4-nemeton-1.html>         |   ✅    |
 | [🔥STAVR🔥](https://github.com/obajay)     | <https://github.com/obajay/StateSync-snapshots/tree/main/Projects/OKP4#snapshot-04-gb-updated-every-5-hours>  |   ✅    |
+| [AutoStake](https://autostake.com)    | <https://snapshots.autostake.com/okp4-nemeton-1>   | ✅      |
 
 </details>
 
 <details>
-  <summary>State / Quick Sync</summary>
+  <summ>ary>State / Quick Sync</summary>
```

Comment:
```suggestion
  <summary>State / Quick Sync</summary>
```

Can you please revert this change ? :)

Reply:
Interesting thanks, not sure how that happened